app/src/androidTest/java/com/lightricks/feedexercise/data/callFeedRepoTests.kt

```diff
@@ -2,37 +2,32 @@ package com.lightricks.feedexercise.data
 
 import androidx.room.Room
 import androidx.test.core.app.ApplicationProvider
+import com.lightricks.feedexercise.data.Constans.hardCodedList
 import com.lightricks.feedexercise.database.FeedDatabase
-import com.lightricks.feedexercise.database.UserProject
 import org.junit.Before
 import org.junit.Test
 
 class callFeedRepoTests {
```

Comment:
```suggestion
class CallFeedRepoTests {
```

Reply:
Why are there both `FeedRepositoryTest` and `CallFeedRepoTests`?

Reply:
so that i can pass params (such as different mock FeedApiServices), thats how I did it in the first exercise, is there a better way to do it?